main/res/layout/cacheslist_item.xml

```diff
@@ -108,7 +108,7 @@
         <view
             android:id="@+id/direction"
             android:layout_width="78dip"
-            android:layout_height="28px"
+            android:layout_height="28dip"
             android:layout_centerHorizontal="true"
             android:layout_gravity="center"
             android:layout_marginTop="21dip"
```

Comment:
A bit unrelated to your PR but might be interesting for you:

There are two primary layout groups used in cgeo:
- Linear Layout
- Relative layout

In a linear layout all elements are added below each other. So no need to calculate the position manually, but child element can't overlap. 

In a relative layout you are way more flexible and can position elements everywhere you want relative to the view group size. There are parameters like `layout_alignParentBottom`, `layout_below`, `layout_marginTop` and much more. However RelativeLayouts are more komplex to write and take longer to inflate on the end device.

In this case, it seems to me the RelativeLayout is used to simulate a vertical Listview, which is IMHO neither nice to read nor good for performance reasons. If I would have written the code myself, I would have used a LinearLayout instead. If you have time and want to play around a bit more, you could try to refactor the "direction and distance" and "inventory and favorites" layout to a LinearLayout. But if you don't want to tackle this, it's totally fine as well as it had worked the last years with a RelativeLayout as well ;-)

Reply:
Thanks for this explanation!!